doc/source/users_guide/index.rst

```diff
@@ -45,10 +45,10 @@ The following example demonstrates how to display a mesh with and without edges:
 
 .. code-block:: python
 
-    from ansys.fluent.visualization import GraphicsWindow, Mesh
+    from ansys.fluent.visualization import VisualizerWindow, Mesh
 
     mesh = Mesh(solver=solver_session, show_edges=True, surfaces=["in1", "in2", "in3"])
-    window = GraphicsWindow(grid=(1, 2))
+    window = VisualizerWindow()
```

Comment:
@prmukherj 
This rename seems a bit obscure.

Reply:
Reverted.